packages/vue/src/components/organisms/SfTable/SfTable.stories.js

```diff
@@ -0,0 +1,46 @@
+import { storiesOf } from "@storybook/vue";
+import { withKnobs } from "@storybook/addon-knobs";
+
+import SfTable from "./SfTable.vue";
+import SfButton from "../../atoms/SfButton/SfButton.vue";
+
+storiesOf("Organisms/Table", module)
+  .addDecorator(withKnobs)
+  .add("Default", () => ({
+    data() {
+      return {
+        headers: [
+          "Quantity",
+          "Payment date",
+          "Payment method",
+          "Amount",
+          "Status"
+        ],
+        content: [
+          ["#35767", "4th Nov", "Paypal", "12.00$"],
+          ["#35767", "4th Nov", "Visa", "15.00$"],
+          ["#35767", "4th Nov", "Paypal", "12.00$"]
+        ]
+      };
+    },
+    components: { SfTable, SfButton },
+    template: `<SfTable>
+    <template>
+      <SfTableRow class="sf-table-row--header">
```

Comment:
Wouldn't it be wiser to have separate `SfTableHeader` component? `is-header` prop and adidng additional CSS rules doesn't seem like a good developer experience